internals/core_models.py

```diff
@@ -88,6 +88,7 @@ class FeatureEntry(ndb.Model):  # Copy from Feature
   name = ndb.StringProperty(required=True)
   summary = ndb.TextProperty(required=True)
   category = ndb.IntegerProperty(required=True)
+  enterprise_product_category = ndb.IntegerProperty(required=True, default=ENTERPRISE_PRODUCT_CATEGORY_CHROME_BROWSER_UPDATE)
```

Comment:
Could this not be required?  It has a default defined for when a value is not specified.